src/main/java/hudson/remoting/MimicException.java

```diff
@@ -29,7 +33,8 @@ public String toString() {
         return (message != null) ? (s + ": " + message) : s;
     }
 
-    public static Throwable make(Channel ch, Throwable cause) {
+    @Nullable
```

Comment:
`@Nullable` can cause confusion for a lot of people, any reason you wanted to avoid `@CheckForNull`... the contract of this method is that the nullability of the result follows the nullability of the provided parameter... so callers should be aware of whether they pass in a null... not unreasonable to ask them to be aware of the potential to get a null back when providing one... smells like a :bug: